src/Compilers/Core/RebuildTest/CSharpRebuildTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Linq;
+using System.Reflection.PortableExecutable;
+using BuildValidator;
+using Castle.Core.Logging;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Emit;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Microsoft.Extensions.Logging;
+using Xunit;
+
+namespace Rebuild.UnitTests
+{
+    //[CompilerTrait(CompilerFeature.)]
+    public class CSharpRebuildTests : CSharpTestBase
+    {
+        [Theory]
+        [InlineData(Platform.Arm64)]
+        [InlineData(Platform.Arm)]
+        [InlineData(Platform.X64)]
+        [InlineData(Platform.Itanium)]
+        [InlineData(Platform.X86)]
+        [InlineData(Platform.AnyCpu)]
+        [InlineData(Platform.AnyCpu32BitPreferred)]
```

Comment:
This approach + not using `Platform.ToString()` gives us a bit of a gap. Imagine in the future that we add a new value to `Platform`. What will trigger to make us update `PlatformFacts` to handle the new entry?

Reply:
Fixed